config-base.js

```diff
@@ -25,37 +28,48 @@ module.exports = {
       plugins: ['@typescript-eslint'],
       parser: '@typescript-eslint/parser',
       parserOptions: {
-        ecmaVersion: 2021,
+        ecmaVersion: 2022,
         sourceType: 'module',
         project: true,
         warnOnUnsupportedTypeScriptVersion: true,
       },
-      // if adding a typescript-eslint version of an existing ESLint rule make sure to disable the ESLint rule here
+      /**
+       * Global overrides.
+       * if adding a typescript-eslint version of an existing ESLint rule,
+       * make sure to disable the ESLint rule here
```

Comment:
was having trouble working in here so below ill comment on net new lines - mostly just organized between off, warn, and error